lms/djangoapps/grades/models.py

```diff
@@ -423,8 +405,6 @@ def bulk_create_grades(cls, grade_params_iter, course_key):
         for params in grade_params_iter:
             cls._prepare_attempted_for_create(params, first_attempt_timestamp)
         grades = [PersistentSubsectionGrade(**params) for params in grade_params_iter]
-        for grade in grades:
-            grade.full_clean()
```

Comment:
This particular call to `full_clean` is the one that was destroying bulk creation query counts.